### ISSUE TITLE
Segovia Demo Sample Set from SP can not be loaded

### ISSUE THREAD

**Issue by oleg68** [GrandOrgue/OdfEdit#46]
After converting [Segovia Sample Set Demo](https://www.sonusparadisi.cz/en/blog/echevarria-virtual-organ-segovia-sample-set/) from Sonus Paradisi with OdfEdit 2.9 when I'm trying to load this organ with GrandOrgue I receive an error: ``Out of range value at section 'Rank084' entry FirstMidiNoteNumber: 999``

The generated ODF file contains such section:

```
[Rank084]
Name=Front A (direct): Blower noise UNUSED
FirstMidiNoteNumber=999
NumberOfLogicalPipes=-998
WindchestGroup=1
Percussive=N
```

that is totally invalid.

**Comment by eturpault** [comment]
Fixed in OdfEdit v2.10.

**Comment by oleg68** [comment]
Seems it works.
